Shopify/cli
# Own polaris modal

### WHY are these changes introduced?
The Dev console uses Polaris, but Polaris is for admin only and it.  Here we take ownership of the last Polaris dependencies and then remove Polaris.

### WHAT is this pull request doing?

1. Copy paste the Polaris Modal into our own codebase.
2. Simplify the implementation to match our simpler requirements
3. Replace instances of the Polaris Modal with our new Modal
4. Remove Polaris
5. Fix tests

### How to test your changes?

1. `pnpm install && pnpm build`
2. cd fixtures/app
3. pnpm dev
4. Open the QR Code Modal. 
5. Open the Post Purchase Modal.
6. Do you notice anything funky?

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
